docs/user_peripherals/07_CAN_info.md

```diff
@@ -0,0 +1,75 @@
+<!---
+
+This file is used to generate your project datasheet. Please fill in the information below and delete any unused
+sections.
+
+You can also include images in this folder and reference them in the markdown. Each image must be less than
+512 kb in size, and the combined size of all images must be less than 1 MB.
+-->
+
+## What it does
```

Comment:
Please could you add title, author and peripheral index above this, as in the latest version of the template: https://github.com/TinyTapeout/tinyqv-full-peripheral-template/blob/main/docs/info.md